WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift

```diff
@@ -21,9 +21,9 @@ final class QuickStartPromptViewController: UIViewController {
 
     private let blog: Blog
 
-    /// Closure to be executed upon dismissal.
+    /// Closure to be executed upon dismissing the Login Epilogue.
     ///
-    var onDismiss: ((Blog) -> Void)?
+    var onDismissEpilogue: (() -> Void)?
```

Comment:
I feel like `onDismiss` might be a better name here, as this screen doesn't really have anything to do specifically with the login epilogue and could in theory be used in different places too.